src/core/core.tooltip.js

```diff
@@ -87,6 +87,9 @@ module.exports = function(Chart) {
 					backgroundColor: view.backgroundColor
 				};
 			},
+			textLabelColor: function(tooltipItem, chart) {
+				return chart.tooltip._options.bodyFontColor;
```

Comment:
i have updated the code using `this`. 
since we need to pass the chart object, how to bypass the eslint error ?